app/_data/tables/version_errors_konnect.yml

```diff
@@ -1709,3 +1790,87 @@ messages:
+      `config.ttl`, `config.neg_ttl`, `config.resurrect_ttl` but Kong Gateway versions
+      < 3.4.0.0 do not support these fields.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.4.0.0` or above.
+    DocumentationURL: |
+      [Vaults entity](/gateway/latest/admin-api/#vaults-object)
+  - ID: P185
+    Severity: error
+    Description: |
+      For the `rate-limiting` plugin, one or more of the following `config` fields are
+      set: `sync_rate` but Kong Gateway versions < 3.4.0.0 do not support these
+      fields. Plugin features that rely on these fields are not working as intended.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.4.0.0` or above.
+    DocumentationURL: |
+      [Rate Limiting plugin](/hub/kong-inc/rate-limiting/)
+  - ID: P186
+    Severity: error
+    Description: |
+      For the `opentelemetry` plugin, the config field `header_type` has been set to
+      `aws`. This is not supported in Kong Gateway versions < 3.4.0.0. Impacted fields
+      have been updated to their default values: `header_type=preserve`
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.4.0.0` or above.
+    DocumentationURL: |
+      [OpenTelemetry plugin](/hub/kong-inc/opentelemetry/)
+  - ID: P187
+    Severity: error
+    Description: |
+      For the `zipkin` plugin, one or more of the fields `header_type`,
+      `default_header_type` has been set to `aws`. This is not supported in Kong
+      Gateway versions < 3.4.0.0. Impacted fields have been updated to their default
+      values: `header_type=preserve` and/or `default_header_type=b3`
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.4.0.0` or above.
+    DocumentationURL: |
+      [Zipkin plugin](/hub/kong-inc/zipkin/)
+  - ID: P188
+    Severity: error
+    Description: |
+      For the `ip-restriction` plugin, the'protocols` array contains one or both of:
+      `TLS`, `TCP`.This is not supported in Kong Gateway versions < 3.4.0.0. These
+      values have been removed and plugin features that rely on these values are not
+      working as intended. If `TLS` and/or `TCP` were the only values, then protocols
+      has been reset to the default values of [`HTTP`, `HTTPS`, `GRPC`, `GRPCS`].
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.4.0.0` or above.
+    DocumentationURL: |
+      [IP Restriction plugin](/hub/kong-inc/ip-restriction/)
+  - ID: P189
+    Severity: error
+    Description: |
+      Kong Gateway version only supports the wRPC protocol which has been removed from
+      Kong Gateway >= 3.2.0.0. Konnect will be removing support for these releases of
+      Kong Gateway on 2023/09/30. After this date these versions of Kong Gateway will
+      not be able to establish connections with Konnect to retrieve configuration
+      changes. Kong recommends upgrading to Kong Gateway 3.4.0.0 LTS.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.4.0.0` or above.
```

Comment:
Do we have a docs link for this?

Reply:
We don't, and I wouldn't know what to link to. This is really only mentioned in our changelog. Maybe just put N/A?

Reply:
Okay to leave blank or should missing documentation links always include "N/A"?

Reply:
Would've been okay to leave blank, or add like a `-` character. We _usually_ like to put some placeholder though so that it doesn't look like a mistake.